docs/rocksdb-indexes.md

```diff
@@ -0,0 +1,53 @@
+# Summary
+
```

Comment:
Do you think it makes sense to add a link to the original issue?

It has some discussions that could yield more info